apps/worker/app/routers/runs.py

```
@@ -0,0 +1,186 @@
+    run_id = str(uuid.uuid4())
+
+    try:
+        # Create database connection
+        conn = get_db_connection()
+        cursor = conn.cursor()
+
+        # Validate that flow exists
+        cursor.execute("SELECT id FROM flows WHERE id = ?", (request.flow_id,))
+        flow = cursor.fetchone()
+        if not flow:
+            raise HTTPException(status_code=400, detail="Flow not found")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)